src/timeline/types.ts

```diff
@@ -1,17 +1,24 @@
+import { PillProps, PillType } from "../pill";
+
 export type Variant =
     | "completed"
     | "current"
     | "upcoming-active"
     | "upcoming-inactive";
 
-export interface TimelineStatusProps {
-    type: "dark" | "light";
-    label: string;
+export type TimelineStatusType = PillType | "dark" | "light";
+
+/** @deprecated This component is deprecated, use PillProps instead */
+export interface TimelineStatusProps extends Omit<PillProps, "type"> {
+    type: TimelineStatusType;
+    /** @deprecated This field is deprecated */
+    label?: string;
 }
 
 export interface TimelineItemProps {
     title: string | JSX.Element;
     content: string | JSX.Element;
+    // statuses?: PillProps[] | undefined;
```

Comment:
Remove commented code

Reply:
Was leaving it here for future change when TimelineStatusType fully removed, guessed I should have make clearer comment. But ya I will remove it.